interface/forms/LBF/new.php

```diff
@@ -1033,8 +1014,8 @@ function warehouse_changed(sel) {
 
                     if ($subtitle) {
                         // There is a group subtitle so show it.
-                        echo "<tr><td class='bold' style='color:#0000ff' colspan='" . attr($CPR) . "'>" . text($subtitle) . "</td></tr>\n";
-                        echo "<tr><td class='bold' style='height:4pt' colspan='" . attr($CPR) . "'></td></tr>\n";
+                        echo "<tr><td class='bold border-top-0' style='color:#0000ff' colspan='" . attr($CPR) . "'>" . text($subtitle) . "</td></tr>\n";
+                        echo "<tr><td class='bold border-top-0' style='height:4pt' colspan='" . attr($CPR) . "'></td></tr>\n";
```

Comment:
In this entire file replace `bold` with `font-weight-bold` as the `bold` class is deprecated. Also convert the height to rem instead of pt (In this case would be (4 x 1.3) x 0.0625 = 0.3125rem).